api/userstore/client.go

```diff
@@ -26,6 +28,17 @@ func New(api restapi.Connector) *UserStore {
 	return &UserStore{api: api}
 }
 
+// LocalUser returns details about the local user
+func (store *UserStore) LocalUser(id string) (user *LocalUser, err error) {
```

Comment:
Would it make sense to keep the Get, Create, Update and Delete for the function names? Then it is clear what the function does.

Reply:
Usage of Get is against best Golang practice: https://golang.org/doc/effective_go#Getters but all in all this your call. May be this is a time to make decision and alight it everywhere.